src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js

```diff
@@ -5,6 +5,10 @@ import { Loading } from '../../Loading/Loading';
 import * as S from './ReportsTrafficSources.styles';
 import { BoxMessage } from '../../styles/messages';
 
+const SafeDivide = (number, quantity) => {
+  return quantity ? number / quantity : 0;
+};
```

Comment:
Nice! since this is a very simple function you can rewrite it in this way:

```js
const SafeDivide = (number, quantity) => quantity ? number / quantity : 0;
```